### ISSUE TITLE
Plans for a new release?

### ISSUE THREAD

**Issue by msilpala** [ContainX/openstack4j#1207]
Greetings.

I wonder whether there are plans for another public release of openstack4j any time soon? We have an issue where production is throwing exceptions when parsing OST `/images` responses. Specifically it expects the `locations` property to be an array of strings whereas OST is actually returning an array of JSON objects.

It appears that [this commit](https://github.com/ContainX/openstack4j/commit/bb78146aa914b855c2d77e2b4d42e455190fb8eb) addresses the issue. Even though that commit is many months old now, there hasn't yet been a tagged release which includes it.

For more context, we are using [spinnaker/clouddriver](https://github.com/spinnaker/clouddriver), which includes openstack4j as a transitive dependency.

If there is a new release expected soon, we may be able to hold tight for it. If not, we need to begin resolving it in a different way.

Thanks much!

@gondor @auhlig

**Comment by varikin** [comment]
@gondor @auhlig Have you had a chance to look at this?

**Comment by varikin** [comment]
@gondor @auhlig Can you please make a new release?

**Comment by vadsiva** [comment]
I'm afraid that there will be no new releases. Project owner @gondor has lost all interest in it.

**Comment by ayberkcal** [comment]
> I'm afraid that there will be no new releases. Project owner @gondor has lost all interest in it.

Yes, looks @gondor has lost interest but this is an opensource project. Everybody can fork and look forward.

**Comment by gondor** [comment]
OpenStack4j 3.2.0 Released - 
https://github.com/ContainX/openstack4j/releases/tag/3.2.0

**Comment by gondor** [comment]
@vadsiva @ayberkcal @varikin - see comment above.  I'm still around and will be more available to review, release and maintain going forward.  

Some quick insights:

- I've moved and became passionate in Golang and other languages so did kind of lose interest.  Not in the project itself but the language
- I've worked for Silicon Valley startups for years and for the past few years haven't been with any company using OpenStack.  It was also too much work to run it locally so it made things more challenging.  
- I was hoping the community at the time I was really active would contribute more.  It was consuming a ton of my time to do most the heavy lifting.

I'm all in favor of picking up momentum and being a good maintainer If the community can get more involved, contribute, test, document.  If someone also has an instance I can use for testing, I would even jump back into contributing code :). 

Hope that helps!
